src/pages/RFOX/hooks/useEpochHistoryQuery.ts

```diff
@@ -12,7 +12,9 @@ import {
 
 type EpochHistoryQueryKey = ['epochHistory']
 
-const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs('2024-07-01T00:00:00Z').unix())
+// TODO(gomes): revert me -  this obviously won't work until first rewards epoch start
+// const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs('2024-07-01T00:00:00Z').unix())
+const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs().subtract(1, 'month').unix())
```

Comment:
i.e we can't have a start timestamp in the future, as this guy will throw https://github.com/shapeshift/web/blob/2a1c9d59b541b87a665e312b16e37e7e3a82ce95/src/pages/RFOX/hooks/useEarliestBlockNumberByTimestampQuery/useEarliestBlockNumberByTimestampQuery.ts#L43